src/index.css

```diff
@@ -86,18 +102,34 @@ main {
   display: flex;
   flex-direction: column;
   gap: 15px;
-  max-width: 250px;
 }
 
 .submit-score {
-  align-self: flex-end;
+  align-self: center;
+  width: 30%;
+  height: 35px;
 }
 
 .score-form input {
-  height: 20px;
+  height: 30px;
   border: 3px solid black;
 }
```

Comment:
- [ Optional] Your Leaderboard looks great, however, I recommend adding a margin to the score Inputs so that it will look more professional.